caffe_app/consumers.py

```diff
@@ -90,103 +110,17 @@ def ws_receive(message):
             export_keras_json.delay(net, net_name, True, reply_channel)
 
     elif (action == 'UpdateHighlight'):
-        add_highlight_to = data['addHighlightTo']
-        remove_highlight_from = data['removeHighlightFrom']
-        user_id = data['userId']
-        highlight_color = data['highlightColor']
-        username = data['username']
-
-        Group('model-{0}'.format(networkId)).send({
-            'text': json.dumps({
-                'addHighlightTo': add_highlight_to,
-                'removeHighlightFrom': remove_highlight_from,
-                'userId': user_id,
-                'action': action,
-                'randomId': randomId,
-                'highlightColor': highlight_color,
-                'username': username
-            })
-        })
-    else:
-        # save changes to database to maintain consistency
+        group_data = update_data(['addHighlightTo', 'removeHighlightFrom', 'userId', 'highlightColor', 'username'])[1]
```

Comment:
Can you create an array before call and use that array as a parameter to the call.